ocaml/lambda/lambda.ml

```diff
@@ -938,21 +938,6 @@ let next_raise_count () =
 (* Anticipated staticraise, for guards *)
 let staticfail = Lstaticraise (0,[])
```

Comment:
Can we delete this from this module and instead inline it at the callsite? The exact identity of this doesn't matter so much anymore -- now we just care that it noisily raises if it escapes translation.

Reply:
Makes sense, done!